source/client/output_formatter_impl.cc

```diff
@@ -157,6 +157,100 @@ YamlOutputFormatterImpl::formatProto(const nighthawk::client::Output& output) co
   return Envoy::MessageUtil::getYamlStringFromMessage(output, true, true);
 }
 
+absl::StatusOr<std::string>
+CsvOutputFormatterImpl::formatProto(const nighthawk::client::Output& output) const {
+
+  std::stringstream ss;
+  ss << "Nighthawk - A layer 7 protocol benchmarking tool." << std::endl << std::endl;
+  for (const auto& result : output.results()) {
+    if (result.name() == "global") {
+      for (const auto& statistic : result.statistics()) {
```

Comment:
Here and below where the `auto` keyword hides the type from the reader, can we spell the type out to help with understanding of the code?

https://google.github.io/styleguide/cppguide.html#Type_deduction